Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift

```diff
@@ -51,6 +51,86 @@ final class SimplifiedLoginManagerTests: XCTestCase {
         self.wait(for: [expectation], timeout: 2)
     }
     
+    func testRequestSimplifiedLogin_failingFetcher() {
+        //Mock Client
+        let mockHTTPClient = MockHTTPClient()
+        let jwks = RemoteJWKS(jwksURI: Fixtures.clientConfig.serverURL.appendingPathComponent("/oauth/jwks"), httpClient: mockHTTPClient)
+        let tokenHandler = TokenHandler(configuration: Fixtures.clientConfig, httpClient: mockHTTPClient, jwks: jwks)
+        let mockClient = MockClient(configuration: Fixtures.clientConfig,
+                                      sessionStorage: MockSessionStorage(),
+                                      stateStorage: StateStorage(storage: MockStorage()),
+                                      httpClient: mockHTTPClient,
+                                      jwks: jwks,
+                                      tokenHandler: tokenHandler)
+        let returnedUserSession = UserSession(clientId: "aClientId", userTokens: Fixtures.userTokens, updatedAt: Date())
+        mockClient.userSessionToReturn = returnedUserSession
+        
+        // Mock Fetcher
+        let mockFetcher = MockSimplifiedLoginFetcher()
+        mockFetcher.fetchDataToReturn = .failure(LoginError.canceled)
+        
+        // Visible Window
+        let vc = UIViewController()
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        window.rootViewController = vc
+        window.makeKeyAndVisible()
+        
+        let sut = SimplifiedLoginManager(client: mockClient, contextProvider: ASWebAuthSessionContextProvider(), env: .pre, completion: { result in }, fetcher: mockFetcher)
```

Comment:
I mentioned it in your previous PR. The completion handler should be the last argument